### ISSUE TITLE
Bundling not currently supported

### ISSUE THREAD

**Issue by guybedford** [screendriver/plugin-sass#3]
When I try to bundle I get an error in the fetch implementation, which doesn't seem like it is designed to run in the browser. This probably needs to be subbed for an fs implementation. Just let me know if you need any help with this at all.

**Comment by robertbaker** [comment]
+1 Must have bundling support.

**Comment by screendriver** [comment]
Hmm I am using the shiny new [fetch API](https://fetch.spec.whatwg.org). Because this is currently not supported in all browsers I use this [polyfill](https://github.com/github/fetch). The polyfill is designed to run **only** in the browser. @guybedford could you please help me with that? I don't know why I get `self is not defined`. If I run the code that should be bundled directly in the browser everything works fine.

```sh
❯ jspm bundle test/bundle-test.js test/bundle.js
     Building the bundle tree for test/bundle-test.js...

/home/me/development/plugin-sass/node_modules/jspm/node_modules/systemjs/dist/system.src.js:1262
        throw addToError(e.toString(), 'Evaluating ' + load.address);
              ^

err  ReferenceError: file:///home/me/development/plugin-sass/jspm_packages/npm/whatwg-fetch@0.9.0/fetch.js:6
       if (self.fetch) {
      ^
     self is not defined
	Evaluating file:///home/me/development/plugin-sass/jspm_packages/npm/whatwg-fetch@0.9.0/fetch.js
	Evaluating file:///home/me/development/plugin-sass/jspm_packages/npm/whatwg-fetch@0.9.0.js
	Error loading file:///home/me/development/plugin-sass/src/sass.js
```

**Comment by screendriver** [comment]
Today I tried it once again and now it seems to work magically :hushed:
Maybe there was an update on a library or something?

I will close this for now but feel free to reopen it when this error happens again.

**Comment by screendriver** [comment]
I have to reopen this issue.

Bundling is not supported in the following case: the SCCS file(s) should be converted to a static CSS file and this CSS file should be included in the bundle so that there is no need for heavy SASS compiling at runtime in the browser. Runtime compiling should only be needed for testing / development.

**Comment by screendriver** [comment]
Fixed with 6a4829b6142e9bf6dfa1f1b67462542b9f51db3c
